src/rail/estimation/algos/deepdisc.py

```diff
@@ -1,4 +1,31 @@
+import numpy as np
 from rail.estimation.estimator import CatEstimator, CatInformer
+from rail.core.data import TableHandle
+from rail.core.common_params import SHARED_PARAMS
+from detectron2.config import LazyConfig, get_cfg
```

Comment:
A lot of these imports are actually being used right now. I'm assuming that this block was mostly copied over from another location. I can take a pass over this and clear out the imports that aren't used if you're ok with it.

Reply:
That would be great!